sdk/scheduler/src/test/resources/ranges.yml

```diff
@@ -0,0 +1,24 @@
+name: "ranges"
+pods:
+  meta-data:
+    count: 2
+    tasks:
+      some-task:
+        goal: RUNNING
+        cmd: "some command"
+        ports:
+          name1:
+            port: 0
+            env-key: key1
+            ranges:
+              - begin: 1
+                end: 21
+              - begin: 2000
+                end: 5050
```

Comment:
Does this mean both port `2000` and `5050` are allowed as well or is this a left closed, right open interval ? Is this documented somewhere (or are there any plans to update docs) ?

Reply:
2000 and 5050 are allowed as well, leaving begin or end empty defines an open interval